src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -9,16 +14,28 @@ import grizzled.slf4j.Logging
 // Event deletion
 
 class Client(
-  config: Config = ConfigFactory.load(),
-  // These will move to a config file:
-  val scheme: String = "https",
-  val authority: String = "api.keen.io",
-  val version: String = "3.0"
+    config: Config = ConfigFactory.load(),
+    // These will move to a config file:
+    val scheme: String = "https",
+    val authority: String = "api.keen.io",
+    val version: String = "3.0"
   ) extends HttpAdapterComponent with Logging {
 
+  // constants
+  val MinSendInterval: Integer = 60
```

Comment:
I think that all of the configuration related to batching should go in the Writer, rather than the Client because it does not apply to all clients.